packages/table/src/common/classes.ts

```diff
@@ -44,6 +45,8 @@ export const TABLE_OVERLAY_LAYER = "bp-table-overlay-layer";
 export const TABLE_POPOVER_WHITESPACE_NORMAL = "bp-table-popover-whitespace-normal";
 export const TABLE_POPOVER_WHITESPACE_PRE = "bp-table-popover-whitespace-pre";
 export const TABLE_REGION = "bp-table-region";
+export const TABLE_REORDERING = "bp-table-reordering";
+export const TABLE_REORDERING_CURSOR_SHIELD = "bp-table-reordering-cursor-shield";
```

Comment:
What's a shield? Is there a more consistent term we could use from the already existing components, such as "indicator" or something?